pygmt/base_plotting.py

```diff
@@ -1922,3 +1922,240 @@ def update_pointers(data_pointers):
             with file_context as fname:
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("meca", arg_str)
+
+    @fmt_docstring
+    @use_alias(
+        A="vector",
+        B="frame",
+        C="cmap",
+        D="rescale",
+        E="uncertainty_color",
+        G="facecolor",
+        J="projection",
+        L="line",
+        N="no_clip",
+        R="region",
+        S="scaling",
+        U="timestamp",
+        V="verbose",
+        W="pen",
+        X="xshift",
+        Y="yshift",
+        p="perspective",
+        t="transparency",
+    )
+    @kwargs_to_strings(R="sequence", i="sequence_comma")
+    def velo(self, data=None, scaling=None, **kwargs):
+        """
+        Plot velocity vectors, crosses, and wedges
+
+        Reads data values from files, numpy array or pandas DataFrame and will
+        plot velocity arrows on a map. Most options are the same as for plot,
+        except *scaling*.
+
+        Must provide  *data* and *scaling*.
+
+
+        Full option list at :gmt-docs:`supplements/geodesy/velo.html`
+
+        {aliases}
+
+        Parameters
+        ----------
+        data : str or pandas.DataFrame
+            Pass in either a file name, a 2D numpy array, or a pandas dataframe
+            table.
```

Comment:
```suggestion
        data : str or np.ndarray or pandas.DataFrame
            Pass in either a file name, a 2D numpy array, or a pandas dataframe
            table. Note that text columns are only supported with file or
            pandas dataframe inputs.
```
Sorry for asking you to change this back! Also added an extra comment to mention that `numpy.ndarray` inputs are only supported if no text columns are provided!

Reply:
> Sorry for asking you to change this back! Also added an extra comment to mention that `numpy.ndarray` inputs are only supported if no text columns are provided!

Not problem @weiji14. So if I understand well, I don't have anything to modify in the code? I just add a comment in the docstring about the format of `data` (It can be a `numpy.ndarray` if there is no text columns?)